### PR TITLE
Ct 1085 refactor ci pipeline

### PR DESCRIPTION
**Pull request by alpha-nu**
Cleaning up the commands a bit

### CODE REVIEW

File: buildScripts/compressArtifacts.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+cd /release/build/conductor;
```
Comment (flebel) [inline buildScripts/compressArtifacts.sh]: No need for a trailing `;`.

File: buildScripts/compressArtifacts.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+version=$(jq -r .version ../package.json)
```
Comment (flebel) [inline buildScripts/compressArtifacts.sh]: Might want to wrap this in double quotes.

File: buildScripts/compressArtifacts.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+if [[ $CI_BRANCH != "master" ]]
```
Comment (flebel) [inline buildScripts/compressArtifacts.sh]:
```
if [[ $CI_BRANCH != "master" ]]; then
    releaseSuffix="dev-${CI_BRANCH}-"
fi

```

File: buildScripts/compressArtifacts.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+then releaseSuffix=dev-$CI_BRANCH-
+fi
+
+for artifact in */
```
Comment (flebel) [inline buildScripts/compressArtifacts.sh]: ditto here RE: shell idioms

File: buildScripts/compressArtifacts.sh (hunk 1 above)
Comment (alpha-nu) [reply buildScripts/compressArtifacts.sh]: Done

File: buildScripts/compressArtifacts.sh (hunk 2 above)
Comment (alpha-nu) [reply buildScripts/compressArtifacts.sh]: Done

File: buildScripts/compressArtifacts.sh (hunk 3 above)
Comment (alpha-nu) [reply buildScripts/compressArtifacts.sh]: Done

File: buildScripts/compressArtifacts.sh (hunk 4 above)
Comment (alpha-nu) [reply buildScripts/compressArtifacts.sh]: Done

**Pull request merged**